features/security/src/main/feature/feature.xml

```diff
@@ -125,6 +135,11 @@
         <bundle>mvn:ddf.platform/platform-paxweb-jettyconfig/${project.version}</bundle>
     </feature>
 
+    <feature name="platform-error-servlet" version="${project.version}">
+        <bundle>mvn:ddf.platform.error/platform-error-api/${project.version}</bundle>
+        <bundle>mvn:ddf.platform.error/platform-error-servlet/${project.version}</bundle>
+    </feature>
```

Comment:
❗️ This is duplication of the `error` feature in the utilities `features.xml`: https://github.com/codice/ddf/blob/090b73a83971162a82cfff3686a1c0b06f71725f/features/utilities/src/main/feature/feature.xml#L58 I'd recommend remove this one from this features file and adding the `platform-error-servlet` bundle to the one in the utilities features file. It also would be good to turn that `error` feature into two features for the api and impl.
```
   <feature name="error-api" version="${project.version}"
             description="Platform error handler.">
        <bundle>mvn:ddf.platform.error/platform-error-api/${project.version}</bundle>
        <bundle>mvn:ddf.platform.error/platform-error-servlet/${project.version}</bundle>
        <bundle>mvn:io.fastjson/boon/${boon.version}</bundle>
    </feature>

    <feature name="error-impl" version="${project.version}"
      description="Platform error handler API">
        <feature>error-api</feature>
        <bundle>mvn:ddf.platform.error/platform-error-impl/${project.version}</bundle>
    </feature>
```
I _think_ that's right, but definitely test it out 😄 
You'll also have to update the `platform-app` feature to point to `error-impl`: https://github.com/codice/ddf/blob/df5bfc0f67263dae413f4850246da1cd5439956d/features/apps/src/main/feature/feature.xml#L70